internal/incremental_page_reader.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/wal-g/wal-g/utility"
 )
 
-// "wi" at the head stands for "wal-g increment"
+// IncrementFileHeader "wi" at the head stands for "wal-g increment"
```

Comment:
IncrementFileHeader contains "wi" at the head which stands for "wal-g increment"